broker/ga4gh/broker/endpoints/repositories.py

```diff
@@ -0,0 +1,287 @@
+import logging
```

Comment:
The comments in here largely apply to the endpoint implementations for the other resources (builds, subscriptions) as well